pkg/cmd/scan.go

```diff
@@ -77,6 +78,9 @@ func NewScanCmd() *cobra.Command {
 			}
 
 			providerVersion, _ := cmd.Flags().GetString("tf-provider-version")
+			if providerVersion == "" {
+				providerVersion = aws.DefaultProviderVersion
```

Comment:
Think of github too, this approach cannot work as we are multi cloud

Reply:
You were too fast for me, i was writing my comment ^^

Reply:
oh you're right, I totally forgot about github 😅